Duffel.ApiClient/Duffel.ApiClient.csproj

```diff
@@ -6,13 +6,13 @@
         <Authors>Duffel Technology Ltd.</Authors>
         <Copyright>Duffel Technology Ltd.</Copyright>
         <Company>Duffel Technology Ltd.</Company>
-        <PackageVersion>0.0.3-alpha</PackageVersion>
+        <PackageVersion>0.0.4</PackageVersion>
```

Comment:
Perhaps time to bump the minor version? 😺 
```suggestion
        <PackageVersion>0.1.0</PackageVersion>
```

Reply:
Yes, I was thinking exactly that 👍